pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -306,3 +310,11 @@ func (tr *TaskRun) IsCancelled() bool {
 func (tr *TaskRun) GetRunKey() string {
 	return fmt.Sprintf("%s/%s/%s", "TaskRun", tr.Namespace, tr.Name)
 }
+
+func GetTaskRunServiceAccountName(taskRun *TaskRun) string {
```

Comment:
Any reason for this over `func (tr *TaskRun) GetServiceAccountName() string {` ? (in the end it's the same but we have others *attached* functions already on `TaskRun` :stuck_out_tongue:)

Reply:
It started elsewhere and migrated here to satisfy a common location.
I wasn't sure how these "helper" methods wanted to live.